addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/connector/RFXComSerialConnector.java

```diff
@@ -120,7 +119,7 @@ public void sendMessage(byte[] data) throws IOException {
             throw new IOException("Not connected sending messages is not possible");
         }
 
-        logger.trace("Send data (len={}): {}", data.length, DatatypeConverter.printHexBinary(data));
+        logger.trace("Send data (len={}): {}", data.length, HexUtils.bytesToHex(data));
```

Comment:
The `HexUtils.bytesToHex()` method is applied every time. Should we encapsulate it by an `if (logger.isTraceEnabled())`?